gips/data/core.py

```diff
@@ -827,7 +831,78 @@ def products2assets(cls, products):
         return set(assets)
 
     @classmethod
+    def query_service(cls, products, tiles, textent, update=False, force=False, grouped=False):
+        """
+        query data service -- product-level interface
+        """
+        if grouped:
+            response = {}
+        else:
+            response  = []
```

Comment:
Agreed that this is less than desirable, but for now I will augment the docstring, as the default list-behaviour is what gips.data uses, and the non-default dict-behaviour is only used in the datahandler.